draft-ietf-openpgp-pqc.md

```diff
@@ -888,6 +888,8 @@ If the pair `AES-256` with `OCB` is not explicitly in the list of the "Preferred
 This is justified since `AES-256` and `OCB` are mandatory to implement.
 If the pair `AES-128` with `OCB` is also implicitly added to the list, it is added after the pair `AES-256` with `OCB`.
 
+When encrypting to an OpenPGP certificate that contains a PQ(/T) key, an implementation MUST NOT infer that the certificate's owner supports v2 SEIPD solely on the basis of the implicitly added AEAD algorithm pairs.
```

Comment:
Since now we are going to remove the possibility of using ML-KEM and any other PQC encryption in v4 keys, isn't the whole topic obsolete now? 

PQC encryption implies a v6 key. A v6 key implies support for v6 PKESK and v2 SEIPD.

Reply:
Technically the decision is not final for another month but yes, that's true. I didn't think of that.

I'll close this PR.